feature/space-entrepreneurship-funding-sources.md

```diff
@@ -5,11 +5,10 @@ menu: true
 order: 4
 ---
 
-We were looking for a directory of funding resources for NewSpace startups and couldn't find one. So we decided to start our own.
+*We were looking for a directory of funding resources for NewSpace startups and couldn't find one. So we decided to start our own.*
 
-This open document has everything we have found so far, but it is definitely not complete. [Please email us about resources we've missed](mailto:groundcontrol@orbitalindex.com) (or [submit a Pull Request on GitHub](https://github.com/orbitalindex/orbital-index))! ❤️
+This open document has everything we've found so far, but it is definitely not complete. [Please email us about resources we've missed](mailto:groundcontrol@orbitalindex.com) (or [submit a Pull Request on GitHub](https://github.com/orbitalindex/orbital-index))! ❤️
 
-> Items marked with a 🚀 are space-specific!
```

Comment:
Moving lower. What I'd really like to see is an ability to temporarily hide all non-space-related items.